test/functional/page_objects/common_page.ts

```diff
@@ -411,9 +411,18 @@ export class CommonPageObject extends FtrService {
    * Clicks cancel button on modal
    * @param overlayWillStay pass in true if your test will show multiple modals in succession
    */
-  async clickCancelOnModal(overlayWillStay = true) {
+  async clickCancelOnModal(overlayWillStay = true, ignorePageLeaveWarning = false) {
```

Comment:
Extracted to allow re-calling similar test flow for both navigation options (cancel | confirm)